src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.model;
+
+import java.util.Arrays;
+
+public class FruitTransaction {
+    private final Operation operation;
+    private final String fruitName;
+    private final int quantity;
+
+    public FruitTransaction(Operation operation, String fruitName, int quantity) {
+        this.operation = operation;
+        this.fruitName = fruitName;
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public String getFruitName() {
+        return fruitName;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public enum Operation {
```

Comment:
The enum 'Operation' should be placed in its own file to follow the best practices for class organization and improve readability.